packages/api/src/routes/sensorRoute.js

```diff
@@ -45,6 +45,7 @@ router.get('/reading/farm/:farm_id', SensorController.getReadingsByFarmId);
 router.post('/reading/invalidate', SensorController.invalidateReadings);
 router.post('/unclaim', SensorController.retireSensor);
 router.get('/:location_id/reading_type', SensorController.getSensorReadingTypes);
+router.get('/:farm_id/reading_types', SensorController.getAllSensorReadingTypes);
```

Comment:
@seinnlee one quick fix, can you make the route `/:farm_id/reading_type` singular to be consistent?